GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -643,7 +645,8 @@ private bool HandlePushOnExit(ref bool isError, FormProcess form)
 
                 page.Buttons.Add(btnPushForce);
 
-                TaskDialogButton result = TaskDialog.ShowDialog(Handle, page);
+                Debug.Assert(owner is not null, "The dialog must be owned by another window! This is a bug, please correct and send a pull request with a fix.");
```

Comment:
Subsequently, this method (`AskForAutoPullOnPushRejectedAction`) is only invoked from `HandlePushOnExit`, so `owner` can never be `null`.